os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -227,13 +227,13 @@ static void up_taskdump(FAR struct tcb_s *tcb, FAR void *arg)
 	/* Dump interesting properties of this task */
 
 #if CONFIG_TASK_NAME_SIZE > 0
-	lldbg("%*s | %5d | %4d | %7lu / %7lu\n", CONFIG_TASK_NAME_SIZE,
+	lldbg("%*s | %5d | %4d | %7lu / %7lu | %p\n", CONFIG_TASK_NAME_SIZE,
```

Comment:
To align with title `STACK ALLOC ADDR`, let's add number like `%10p`.

Reply:
Modified. Thanks.